api.py

```diff
@@ -136,7 +136,11 @@ def post_track_of_day_handler(user_id, track_id):
         print("User "+user_id+" posting "+track_id)
         if get_user(user_id):
             if get_track(track_id):
-                pass
+                rows_updated = create_post({"user_id": user_id, "track_id": track_id, "date_posted": int(round(time.time()*1000))})
+                if rows_updated == -1:
+                    return Response(status=400)
+                else:
+                    return Response(status=200)
             else:
```

Comment:
we want to return a 400 if we dont have the track in our db? Dont we need to instead get the info from Spotify and add it to the db?